helmfile/charts/grafana-ops/templates/backup-dashboard-configmap.yaml

```
@@ -0,0 +1,12 @@
+{{- if .Values.dashboards.backup.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: backupdashboard
```

I would prefer these names to be:
```suggestion
  name: backup-dashboard
```

I think that would make it easier to read. Same with the other dashboard configmaps.

---

Moved all dashboards to one configmap (as Lennart requested) called `grafana-dashboards`.